src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -157,50 +159,76 @@ private static string[] GetDefaultLetterKeyAllLanguagesOnly(LetterKey letter)
-                LetterKey.VK_MINUS => new[] { "~", "‐", "‑", "‒", "—", "―", "⁓", "−", "⸺", "⸻", "∓" },
-                LetterKey.VK_SLASH_ => new[] { "÷", "√" },
-                LetterKey.VK_DIVIDE_ => new[] { "÷", "√" },
-                LetterKey.VK_MULTIPLY_ => new[] { "×", "⋅" },
-                LetterKey.VK_PLUS => new[] { "≤", "≥", "≠", "≈", "≙", "⊕", "⊗", "∓", "≅", "≡" },
+                LetterKey.VK_MINUS => new[] { "‐", "‑", "‒", "—", "―", "⁓", "−", "⸺", "⸻" },
+                _ => Array.Empty<string>(),
+            };
+        }
+
+        // Mathematics
+        private static string[] GetDefaultLetterKeyMATH(LetterKey letter)
+        {
+            string[] characters = letter switch
+            {
+                LetterKey.VK_0 => new[] { "₀", "⁰", "°", "∅", "↉" },
+                LetterKey.VK_1 => new[] { "₁", "¹", "½", "⅓", "¼", "⅕", "⅙", "⅐", "⅛", "⅑", "⅒" },
+                LetterKey.VK_2 => new[] { "₂", "²", "⅔", "⅖", "√" },
+                LetterKey.VK_3 => new[] { "₃", "³", "¾", "⅗", "⅜", "∛" },
+                LetterKey.VK_4 => new[] { "₄", "⁴", "⅘", "∜" },
+                LetterKey.VK_5 => new[] { "₅", "⁵", "⅚", "⅝" },
+                LetterKey.VK_6 => new[] { "₆", "⁶" },
+                LetterKey.VK_7 => new[] { "₇", "⁷", "⅞" },
+                LetterKey.VK_8 => new[] { "₈", "⁸", "∞", "∝" },
+                LetterKey.VK_9 => new[] { "₉", "⁹" },
+                LetterKey.VK_A => new[] { "∀" },
+                LetterKey.VK_C => new[] { "ℂ", "∁" },
+                LetterKey.VK_D => new[] { "∂", "◇" },
+                LetterKey.VK_E => new[] { "∃", "∄", "∈", "∉" },
+                LetterKey.VK_F => new[] { "ƒ" },
+                LetterKey.VK_N => new[] { "ⁿ", "ℕ" },
+                LetterKey.VK_P => new[] { "∏", "⊥", "⊢", "⊬" },
+                LetterKey.VK_Q => new[] { "ℚ" },
+                LetterKey.VK_R => new[] { "ℝ" },
+                LetterKey.VK_S => new[] { "∑", "∫", "∮" },
+                LetterKey.VK_T => new[] { "⊨", "⊭" },
+                LetterKey.VK_X => new[] { "×", "⋅" },
+                LetterKey.VK_Z => new[] { "ℤ" },
+                LetterKey.VK_PLUS => new[] { "≤", "≥", "±", "⇔", "₊", "⁺", "≠", "≅", "≈", "≙", "≜", "≔", "≝", "≡", "⊕", "⊗", "⊙" },
+                LetterKey.VK_COMMA => new[] { "≤", "⇐", "⋃", "⋁", "⊆", "⊂", "⊄", "∌", "∋", "≪" },
+                LetterKey.VK_PERIOD => new[] { "≥", "⇒", "⋂", "⋀", "⊇", "⊃", "⊅", "∉", "∈", "≫" },
+                LetterKey.VK_SLASH_ => new[] { "÷", "‰", "½", "⅓", "⅔", "¼", "¾", "√" },
+                LetterKey.VK_MINUS => new[] { "∓", "~" },
+                LetterKey.VK_MULTIPLY_ => new[] { "×", "⋅", "^", "¬", "∴", "∵" },
+                LetterKey.VK_DIVIDE_ => new[] { "÷", "√", "∠", "∡", "∢", "⦝", "⦜" },
                 _ => Array.Empty<string>(),
             };
+
+            return characters.Union(GetDefaultLetterKeyEL(letter)).ToArray();
```

Comment:
Does the EL list contains only math related characters? If not we should split the EL list into EL math and EL text or something.

Reply:
Hi @htcfreek, `GetDefaultLetterKeyEL` contains 7 characters with tonos (ά έ ή ί ό ύ ώ), which are not commonly used in math. In the original commit, this function was not used and these characters were excluded from the math language. Should I revert the original commit?